### PR TITLE
[datafeeder]: Sum-up page

### PR DESCRIPTION
**Pull request by y-vynohradov**
chore: Add wizard summarize page

### CODE REVIEW

**Comment by fgravin** [comment]
Still red :/

File: libs/editor/src/lib/components/wizard-summarize/wizard-summarize.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        {
+          display: 'tagName2',
+          value: 'tagName2',
+        },
+      ])
+    )
+    window.localStorage.setItem('datepicker', String(new Date().getTime()))
+    window.localStorage.setItem('dropdown', JSON.stringify('10000'))
+    window.localStorage.setItem('description', 'description')
+  })
```
Comment (tonio) [inline libs/editor/src/lib/components/wizard-summarize/wizard-summarize.component.spec.ts:66]: while you're at dealing with localstorage, you should refactor to be able to handle multiples import => store with the id as key ?

Comment (tonio) [reply libs/editor/src/lib/components/wizard-summarize/wizard-summarize.component.spec.ts:66]: storage should be done in a service, implementation being private. If you change implementation then you won't be forced to rewrite all tests.

File: apps/datafeeder/src/app/presentation/pages/summarize-page/summarize-page.component.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
+:host {
+  display: flex;
+  flex: 1;
```
[the remark this answers is not in the document]
Comment (y-vynohradov) [reply apps/datafeeder/src/app/presentation/pages/summarize-page/summarize-page.component.css:3]: Do you mean <app-root class="flex flex-col flex-1"></app-root>? 

But component will be rendered inside root and this required for proper displaying 

Look, it is simular to this

`<app-root class="flex flex-col flex-1">
<app-summirize-page class="flex-1 flex"></app-summirize-page>
</app-root>`

**Pull request merged**